framework/src/test/java/tests/AutoValueTest.java

```diff
@@ -0,0 +1,56 @@
+package tests;
+
+import static org.checkerframework.framework.test.TestConfigurationBuilder.buildDefaultConfiguration;
+
+import com.google.common.collect.ImmutableList;
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+import org.checkerframework.framework.test.*;
+import org.junit.runners.Parameterized.Parameters;
+
+public class AutoValueTest extends CheckerFrameworkPerDirectoryTest {
```

Comment:
@kelloggm do we want these tests and the test data in `returnsrcvr` sub-folders?  Or should we keep everything at the top level, in which case we should rename tests, e.g., to `ReturnsRcvrAutoValueTest`?

Reply:
I think keeping them at the top level is fine, but we should rename them as you suggested. The actual test data should definitely be in a `returnsrcvr` folder, though